timepiece/templates/timepiece/contract/view.html

```diff
@@ -56,6 +64,75 @@
                         ({% widthratio contract.hours_worked contract.contracted_hours 100 %}%)
                     </td>
                 </tr>
+                {% if contract.pre_launch_hours_worked %}
+                    <tr class="error"><th>Pre-Contract Hours Worked</th>
+                        <td>
+                            {{ contract.pre_launch_hours_worked|floatformat:2 }}
+                            ({% widthratio contract.pre_launch_hours_worked contract.contracted_hours 100 %}%)
+                            <a class="btn btn-danger btn-mini" data-toggle="popover"
+                               data-html="true"
+                               title="Precontract Entries"
+                               data-content="
+                                   <table class='table table-bordered'>
+                                      <tr>
+                                        <td class='table'>Project</td>
```

Comment:
As we discussed in person, I think having the Entry's activity and hours here would be beneficial to help the PM/Viewer determine how to handle these outliers